### DISCUSSION TITLE
No View column on google analytics

### DISCUSSION THREAD

**Discussion by mohamedhafizelhaj** [freekmurze/laravel-analytics#383] [Q&A]
Hello,

I am trying to test this package locally. I did everything as mentioned but when comes to the view id I can't find it, actually there is no view column, just account and property.
I did a search and found that I must use a universal property, though it asks for the website URL while I am developing locally. When entering anything and moving forward, it turns out there is no view column also.
So how can I use this package now?

**Comment by DevHoracioRodriguez** [comment]
Hello mohafiz

Take a look on the solution - https://github.com/spatie/laravel-analytics/discussions/374
